app/views/unsubscriptions/confirm_already_unsubscribed.html.erb

```diff
@@ -1,5 +1,5 @@
 <%
-page_title = "You’ve already unsubscribed"
+page_title = t("unsubscriptions.title.confirm_already_unsubscribed")
```

Comment:
Minor: I think we usually do these by view i.e. `unsubscriptions.confirm_already_unsubscribed.title`.